src/test/test-utils.test.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * @license
+ * Copyright (c) 2019 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+import test from 'tape';
```

Comment:
Whats good about tape vs mocha?

Reply:
What I like about `tape`, especially compared to `mocha`, is:
- it is physically smaller, about 1/4 the filesize 1/2 the dependencies
- it has its own only-what's-needed integrated testing/assertions
- no magical global state or DSL; you can run a test file with just the `node my.test.js` and it performs identically as if you use the tape CLI.
- it emits TAP (https://testanything.org/) output, which offers a lot of flexibility for pipelining etc.

I originally was exploring TAP based frameworks with an eye for simpler browser testing scenarios and kind of fell in love with the simplicity of this, especially compared with Mocha+Chai or Jest etc.

Reply:
However, I would prefer an even smaller, better and modules-based version of tape, so hopefully that library will present itself some day and I'll convert to it.